### ISSUE TITLE
Add notifications for play/pause/stop

### ISSUE THREAD

**Issue by tkowalczyk** [tkowalczyk/DevTalkMobile#30]
As all of the Android applications have them

**Comment by tkowalczyk** [comment]
https://blog.xamarin.com/background-audio-streaming-with-xamarin-android/

**Comment by freud** [comment]
Yes, this functionality is similar to something I've expected in a first contact with the app.

Going to recently played episode would be great and for me it would be even in the application main menu. I've looked for this in a first contact with the app.
